docs/DAPP_PERFORMANCE_AUDIT.md

```
@@ -0,0 +1,37 @@
+| :--- | :--- | :--- | :--- | :--- |
+| **LCP** (Largest Contentful Paint) | $\le 2.0\text{ s}$ | $2.5\text{ s}$ | $> 3.0\text{ s}$ | Moto G Power, Fast 3G (1.6 Mbps / 150ms RTT) |
+| **INP** (Interaction to Next Paint) | $\le 100\text{ ms}$ | $150\text{ ms}$ | $> 200\text{ ms}$ | 4x CPU Throttling, Active Input Flow |
+| **CLS** (Cumulative Layout Shift) | $\le 0.05$ | $0.08$ | $> 0.10$ | Layout shift throughout async balance fetches |
+| **Initial JS Chunk Size** | $\le 250\text{ KB}$ | $300\text{ KB}$ | $> 350\text{ KB}$ | Uncompressed parsed JS |
+| **Total Route Bundle** | $\le 1.0\text{ MB}$ | $1.2\text{ MB}$ | $> 1.5\text{ MB}$ | Aggregate client bundles |
+
+---
+
+## 3. Bundle Analysis & Optimization Strategy
+
+### 1. Dynamic Code-Splitting
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)